docs/contributing/Issue-Priorities.md

```diff
@@ -0,0 +1,12 @@
+## Issue priorities
+
+The New XKit repository uses four priority labels for categorizing bug reports. These are, in order from most to least important:
+
+* `P-Critical` - All or most of a commonly-used extension is broken, or one feature is broken across multiple extensions, such as an often-used patches function;
```

Comment:
using a comma splice at the end of the sentence here makes it less readable, since it's supposed to be an additional clarification to the second clause, not a third independent clause. (also where did that semicolon come from? O.o)

Reply:
If you are to properly punctuate bullet lists, you add a semicolon at the end of every bullet. Will fix the wording on the second part though.

Reply:
Never mind, turns out I remembered that wrong; that only applies on bullet points in an ongoing sentence.